src/app/pages/list/action-list/action-list.component.scss

```diff
@@ -0,0 +1,13 @@
+@import '~@pxblue/colors/palette.scss';
+
+pxb-info-list-item {
+    cursor: pointer;
+
+    ::ng-deep &.selected mat-list-item {
```

Comment:
do we need ngdeep here or is there another class we expose for this element that we could use to target it?

Reply:
This is the only way I could style the background color right now. We should probably add a class to the InfoListItem to modify the background color.

Reply:
Is this still needed if we removed the select logic?